entity_client.go

```diff
@@ -53,6 +53,11 @@ type EntityClient interface {
 	// caller must ensure that Set.Size is greater than zero and Set.Op and Set.Id are nonempty
 	// strings.
 	WithSet(Set) EntityClient
+
+	// WithTrace returns a new EntityClient that sends the trace string every request
```

Comment:
"sends the trace string _with/for_ every request"

Reply:
Fixed.